release_notes.md

```diff
@@ -25,6 +25,13 @@ This file tracks release notes for the loop client.
   start script and example `docker-compose` environment as well as
   [documentation on how to use the `regtest` Loop server](https://github.com/lightninglabs/loop/blob/master/regtest/README.md)
   was added too.
+* Two new flags, `--max_sweep_fee` and `--max_sweep_fee_portion`, are added to
+  `loop out`. Users now can specify the max on-chain fee to be paid when
+  sweeping the HTLCs. `max_sweep_fee` specifies the max fee can be used in
+  satoshies, despite the total swap amount. `max_sweep_fee_portion` specifies a
```

Comment:
nit: s/satoshies/satoshis/
Also, do you mean "independent of" instead of "despite"?

Reply:
Yep my bad, now fixed.